### PR TITLE
Navigation contribution Sept 2024

### PR DESCRIPTION
**Pull request by CharlotteDowns**
## What

This is the updated guidance for the 'agile version' of our Navigation contribution

### CODE REVIEW

File: src/patterns/navigate-a-service/index.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+- language selectors, if they work across your entire service
+- search inputs, if they only search within your service
+
+To help users understand what the search input will cover, include ‘Search [your service]’ as placeholder text within the search input.
+
+### Finally, show page-specific elements
+
+Show any elements that only affect the current page after the Service header navigation.
+
+Place any [Breadcrumbs](/components/breadcrumbs/) just before the `<main>` element. Placing them here means that the ‘Skip to main content’ link allows the user to skip all navigation links, including breadcrumbs.
```
Comment (trang-erskine) [inline src/patterns/navigate-a-service/index.md:180]: Something weird is happing with the formatting, '<main>' code syntax is wrong?

File: src/patterns/navigate-a-service/index.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+You can add space between the GOV.UK header and Service navigation to insert important elements that apply to your entire service.
+
+Elements you might add here include:
+
+- a [Phase banner](/components/phase-banner/)
+- an ‘organisation switcher’, or a similar tool that helps users use your service across a set of things
+
+See an example of an [organisation switcher](https://design-patterns.service.justice.gov.uk/components/organisation-switcher/) component in the Ministry of Justice Design System.
+
+We’d like to learn more about elements you might place between the GOV.UK header and Service navigation. Tell us in our discussion space [LINK].
```
Comment (trang-erskine) [inline src/patterns/navigate-a-service/index.md:161]: Missing link

File: src/patterns/navigate-a-service/index.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+However, we’ll keep this under consideration and look for other ways to encourage service teams to adopt the Service navigation to show their service name.
+
+### See more about our decision-making and research
+
+To explain our design rationale and decision-making process, we’ve started a series of design notes and discussion topics in our GitHub discussion space.
+
+We also ask service teams to give us their feedback and share research findings that can help us iterate.
+
+Some discussions we’d like to highlight [LINKS]:
```
Comment (trang-erskine) [inline src/patterns/navigate-a-service/index.md:204]: Missing link

File: src/patterns/navigate-a-service/index.md (hunk 1 above)
Comment (CharlotteDowns) [reply src/patterns/navigate-a-service/index.md:180]: I think this one is fixed now

File: package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "accessible-autocomplete": "^3.0.0",
     "clipboard": "^2.0.11",
-    "govuk-frontend": "^5.5.0",
+    "govuk-frontend": "github:alphagov/govuk-frontend#0865cb4fb",
```
Comment (owenatgov) [inline package.json:39]: Making a comment so that we don't forget that this needs to be removed before merging

**Comment by calvin-lau-sig7** [comment]
> Looks really good - have made some small suggestions for style/clarity

Thanks! I've fixed in a text editor so the changes should be in there now. 😄

**Pull request merged**